incubator/clickhouse/values.yaml

```diff
@@ -0,0 +1,163 @@
+## Pod Annotations
+# podAnnotations: {}
+
+## Statefulset annotations
+# annotations: {}
+
+## Number of ClickHouse nodes
+replicas: 1
+
+## Image settings
+## ref: http://kubernetes.io/docs/user-guide/images/
+image: yandex/clickhouse-server
+imageTag: latest
```

Comment:
The version needs to be a tag for reproducibility. Something like `18.10.3`

Reply:
Agree, fixed